internal/splunk/common.go

```diff
@@ -22,12 +22,15 @@ const (
 	DefaultNameLabel           = "otel.log.name"
 	DefaultSeverityTextLabel   = "otel.log.severity.text"
 	DefaultSeverityNumberLabel = "otel.log.severity.number"
-	HECTokenHeader             = "Splunk"
-	HecTokenLabel              = "com.splunk.hec.access_token" // #nosec
+	HecTokenHeader             = "Splunk"
+	HTTPSplunkChannelHeader    = "X-Splunk-Request-Channel"
```

Comment:
please just add this and don't change the existing constants in this PR so we can review

Reply:
Just a note, the name change was my fault as I was wondering why case was inconsistent. It's okay with me to fix later though if others prefer 👍

Reply:
ah, I missed that the first time around. In any case HEC is an acronym for "HTTP Event Collector" and should be kept in caps, not the other way around. I can open a PR to fix that.